fastlane/Fastfile

```diff
@@ -0,0 +1,174 @@
+fastlane_require 'dotenv'
+fastlane_require 'jwt'
+fastlane_require 'base64'
+fastlane_require 'net/sftp'
+
+default_platform(:android)
+
+branch_name = `git rev-parse --abbrev-ref HEAD`
+build = `git rev-list --count #{branch_name} | tr -d " \t\n\r"`
+build = build.to_i + 1958 # adding 1958 for legacy reasons. Must be in sync with getVersionCode() from build.gradle
+version = get_version_name(
+                gradle_file_path:"build.gradle",
+  	            ext_constant_name:"androidVersionName")
+version = version.delete "'"
+
+platform :android do |options|
+
+  desc "Run all the tests"
+  lane :test do |options|
+    gradle(task: "test")
+  end
+
+  desc "Deploy new version to Google Play and APK Store options: beta:false scheduled:false (default)"
+  lane :deploy do |options|
+    release_note_path_en = "metadata/android/en-US/changelogs/default.txt"
+
+    # use english-change-log for french language too
+    FileUtils.cp(release_note_path_en, "metadata/android/fr-FR/changelogs/default.txt")
+
+    deployToPlaystore(scheduled:options[:scheduled], beta:options[:beta])
+
+    if options[:beta]
+      puts "Skipping deployment to server cause there isn't currently a beta channel"
+      
+      slack(
+        default_payloads: [], # reduce the notification to the minimum
+        message: ":rocket: Successfully deployed #{version} with code #{build} to the Play Store :cryptomator:",
+        payload: {
+          "Changes" => File.read(release_note_path_en)
+        }
+      )
+    else
+      deployToServer(scheduled:options[:scheduled], beta:options[:beta])
+
+      slack(
+        default_payloads: [], # reduce the notification to the minimum
+        message: ":rocket: Successfully deployed #{version} with code #{build} to the Play Store and APK store :cryptomator:",
+        payload: {
+          "Changes" => File.read(release_note_path_en)
+        }
+      )
+    end
+
+  end
+
+  desc "Deploy new version to Playstore"
+  private_lane :deployToPlaystore do |options|
+    deploy_target = "production"
+
+    if options[:beta]
+      deploy_target = "beta"
+    end
+
+    gradle(task: "clean")
+
+    gradle(
+      task: "assemble",
+      build_type: "Release",
+      flavor: "playstore",
+      print_command: false,
+      properties: {
+        "android.injected.signing.store.file" => ENV["SIGNING_KEYSOTRE_PATH"],
+        "android.injected.signing.store.password" => ENV["SIGNING_KEYSOTRE_PASSWORD"],
+        "android.injected.signing.key.alias" => ENV["SIGNING_KEY_ALIAS"],
+        "android.injected.signing.key.password" => ENV["SIGNING_KEY_PASSWORD"],
+      }
+    )
+
+    upload_to_play_store(
+      package_name: 'org.cryptomator',
```

Comment:
```suggestion
```

This shouldn't be needed because you have already defined `package_name("org.cryptomator")` in the `Appfile`.